pylmod/gradebook.py

```diff
@@ -80,9 +115,18 @@ def get_assignments(
         return assignments['data']
 
     def get_assignment_by_name(self, assignment_name, assignments=None):
-        """
-        Get assignment by name; returns assignment ID value (numerical)
+        """get assignment by name
+
+        returns assignment ID value (numerical)
         and assignment dict.
+
+        Args:
+            assignment_name (str): name of assignment
+            assignments (dict): assignments
+
+        Returns:
+            json
```

Comment:
Returns a `dict`, maybe worth grabbing an examplar by making this call and copy the pretty printed data structure here